### PR TITLE
link button added

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: docs/button.html [hunk 1]
```diff
@@ -59,6 +59,21 @@ <h2>Components</h2>
                 </code>
             </pre>
 
+            <div>
+                <p>Link button</p>    
+                <div class="button">
```
Comment (OmKakatkar) [inline docs/button.html:64]: Giving a div a class name of button seems to be misleading.  If possible provide a different name for this class

**Review by OmKakatkar — APPROVED** [review]
Left a comment, do have a look. 
Otherwise, looks good to merge

**Review by lomesshh — APPROVED** [review]
clean code
Looks good to merge

**Review by Kavita539 — COMMENTED** [review]
- Code looks clean and maintained.
- The UI is also clean and minimal
- Thankyou for keeping the changes files minimum, was easy to review.

**Pull request merged**